StantUiIosLibrary/Classes/Cells/ProfileWithImageCell.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ProfileWithImageCell.swift
+//  Pods-StantUiIosLibraryDemo
+//
+//  Created by Alexandre Henrique on 7/3/19.
+//
+
+import UIKit
+
+public class ProfileWithImageCell: UITableViewCell {
+    
+    public static let IDENTIFIER = "ProfileWithImageCellIdentifier"
+    public static var HEIGHT: CGFloat = 56
+    
+    public let titleLabel       = UILabel(frame: CGRect(x: 0, y: 0, width: 210, height: 19))
+    public let descriptionLabel = UILabel(frame: CGRect(x: 0, y: 0, width: 210, height: 14))
+    
+    override public func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public func set(title: String = String(),
+                    description: String = String(),
+                    icon: UIImage? = UIImage(),
+                    iconSize: CGSize = CGSize(width: 35, height: 35),
+                    titleSize: CGFloat = 16,
+                    descriptionSize: CGFloat = 14) {
+        
+        if self.subviews.contains(titleLabel) || self.subviews.contains(descriptionLabel) {
+            titleLabel.removeFromSuperview()
+            descriptionLabel.removeFromSuperview()
+        }
+        
+        let roundedAndFormattedImage = icon?.roundedReframedWith(size: iconSize)
+        
+        self.titleLabel.text       = title
+        self.descriptionLabel.text = description
+        self.imageView?.image      = roundedAndFormattedImage
+        
+        positionElements()
+        setElementsStyle(titleSize: titleSize, descriptionSize: descriptionSize)
```

Comment:
We have a extension for setting label styles. Try to use it :)

Reply:
Done!